### DISCUSSION TITLE
How to package Fuxa into one executable file?

### DISCUSSION THREAD

**Discussion by henjoe** [frangoteam/FUXA#769] [Q&A]
Hi @unocelli will it be possible to package Fuxa into one executable file? 
I actually trying it using the pkg module, but I always got error.

Here is my configuration in the pkg:

```
"pkg": {
    "targets": [
      "node16-win-x64"
    ],
    "files": [
      "**"
    ],
    "assets":[
      "node_modules/@foliojs-fork/linebreak/src/classes.trie",
      "**"
    ]
  }
```

This will create a windows.exe application, but when I tried to run the application this gives me error:
![image](https://user-images.githubusercontent.com/49782655/229268410-dad9c4a7-9e63-4bb9-97de-e7638b1da700.png)

I actually want to share fuxa into another computer without installing any dependencies such as nodejs and etc.

Thanks and regards,
Henjoe

**Comment by j03r6-47-w0rk** [comment]
Hi, 

have you resolved this issue in the meantime, was is possible to create a running executable file?
If yes, could you please provide the steps/commands/configs you have done?

Thanks
